examples/big-bang/virtualservices/gitea.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: gitea
+  namespace: zarf
+spec:
+  gateways:
+    - istio-system/public
+  hosts:
+    - gitea.bigbang.dev
```

Can we use the DOMAIN variable here?
```suggestion
    - gitea.###ZARF_VAR_DOMAIN###
```

---

done